src/auth/auth.module.ts

```diff
@@ -0,0 +1,27 @@
+import { Module } from '@nestjs/common';
+import { JwtModule, JwtService } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { RedisModule } from '@liaoliaots/nestjs-redis';
+import { Env } from 'src/env';
+import { JwtStrategy } from './jwt.strategy';
```

Comment:
PrimathonAICodeReviewer

Consider adding a comment or description for the `JwtStrategy` import to improve code readability and maintainability.

```diff
+ // Import JwtStrategy for JWT authentication strategy
+ import { JwtStrategy } from './jwt.strategy';
```



<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->